packages/lesswrong/server/scripts/generativeModels/coverImage.ts

```diff
@@ -0,0 +1,309 @@
+/* eslint no-console: 0 import/no-deprecated:0 */
+
+import OpenAI from 'openai';
+import axios from 'axios';
+import { Globals, createAdminContext, createMutator } from '../../vulcan-lib/index.ts';
+import Posts from '../../../lib/collections/posts/collection.ts';
+import ReviewWinners from '../../../lib/collections/reviewWinners/collection.ts';
+import ReviewWinnerArts from '../../../lib/collections/reviewWinnerArts/collection.ts';
+import { moveImageToCloudinary } from '../convertImagesToCloudinary.ts';
+import { myMidjourneyAPIKeySetting, reviewArtSlackAPIKeySetting } from '../../../lib/instanceSettings.ts';
+import { getOpenAI } from '../../languageModels/languageModelIntegration.ts';
+import { sleep } from '../../../lib/utils/asyncUtils.ts';
+import shuffle from 'lodash/shuffle';
+import { trace } from '../../../lib/helpers.ts';
+import { filterNonnull } from '../../../lib/utils/typeGuardUtils.ts';
+
+const DEPLOY = false
+
+const myMidjourneyKey = myMidjourneyAPIKeySetting.get()
+if (!myMidjourneyKey) {
```

Comment:
This will cause an error to be thrown on server start-up for anyone who doesn't have this instance setting set, which is probably not the behavior we want